server/paymentProviders/transferwise/index.ts

```diff
@@ -104,6 +104,15 @@ async function quoteExpense(
 ): Promise<QuoteV2> {
   await populateProfileId(connectedAccount);
 
+  const isExistingQuoteValid =
+    expense.data?.quote &&
+    expense.data.quote.paymentOption &&
+    moment.utc().subtract(60, 'seconds').isBefore(expense.data.quote.expirationTime);
```

Comment:
What's the rationale for those 60 seconds?

What is the expirationTime in practice?

Reply:
I'm considering a 60 seconds buffer for the transaction creation and fund to be complete. Basically, the whole process that is sometimes taking 25 seconds and causing the network error in https://github.com/opencollective/opencollective/issues/4963.

The expiration time is related to how long does the Quote you created to fulfill the transaction will be valid.

Reply:
What is a normal value for `expirationTime` in practice?

Reply:
ISO date string.

Reply:
Rephrasing, I just wanted to ask how far it is in the future.